sdk/metric/list.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric
+
+import (
+	"sync/atomic"
+	"unsafe"
+)
+
+func (l *sortedLabels) Len() int {
```

Comment:
I feel like they were cluttering up the main logic. This file is full of trivial list manipulations. If you still think I should move these, happy to.

Reply:
Hmm, I see. Changing these types to other files would be an option? It would be a little strange since these types are specifically used in the SDK but maybe better than a huge `sdk` file. 

I'm fine with you splitting these or keep this way.